kubernetes/linux/main.sh

```diff
@@ -463,7 +468,7 @@ if [ ! -z "$podWithValidContainerId" ]; then
       # update runtime only if its not empty, not null and not startswith docker
       if [ -z "$containerRuntime" -o "$containerRuntime" == null  ]; then
             echo "using default container runtime as $CONTAINER_RUNTIME since got containeRuntime as empty or null"
-      elif [[ $containerRuntime != docker* ]]; then
+      else
             export CONTAINER_RUNTIME=$containerRuntime
```

Comment:
what will happen if the runtime is neither docker or containerd for a hybrid cluster ?

Reply:
> what will happen if the runtime is neither docker or containerd for a hybrid cluster ?
> container runtime will be whatever detected for example CRIO. For switching of parser, the existing logic continue to work, i.e. if the container runtime docker is use docker parse else for all the runtimes, it will be "CRI" .

Reply:
ok. so the only place we use runtime string is to choose the right parser for the logs? As long as we have telemetry (which i think we have) on how many clusters we have running with neither docker nor containerd, we are ok i guess.

Reply:
> ok. so the only place we use runtime string is to choose the right parser for the logs? As long as we have telemetry (which i think we have) on how many clusters we have running with neither docker nor containerd, we are ok i guess.
>  Yes, for parser detection is "docker" vs non-docker and detected runtime used for the telemetry, and there are no changes to either of  these in this PR. Yes,  I think, we are good on this.